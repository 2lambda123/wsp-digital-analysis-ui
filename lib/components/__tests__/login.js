/* global describe, it, expect */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'

import Login from '../login'

describe('Component > Icon', () => {
  it('renders correctly', () => {
    const tree = mount(
      <Login />
    )
    expect(mountToJson(tree)).toMatchSnapshot()
  })
})
