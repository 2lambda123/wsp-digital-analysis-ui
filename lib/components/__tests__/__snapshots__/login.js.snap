// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > Icon renders correctly 1`] = `null`;
